.changeset/nervous-dragons-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+'sku': patch
+---
+
+Update deps
```

Comment:
Adding this changeset now, but going to do a few more dependency update PRs before actually releasing it.